spec/lib/wifi_user/use_cases/email_signup_spec.rb

```diff
@@ -100,7 +109,7 @@
         end
 
         it "does not send an email to Notify" do
```

Comment:
This test should probably be renamed to show that we do send a message to Notify now

Reply:
Good spot, I've renamed the test.